ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -127,7 +151,47 @@ class ServicesHomeScreen extends React.Component<Props, State> {
               </Text>
             </TabHeading>
           }
-        />
+        >
+          <ServicesLocal
+            animated={{
+              onScroll: Animated.event(
```

Comment:
Should **onScroll handler** be defined one time and returned from a given index?